pkg/cfn/builder/cluster.go

```diff
@@ -320,7 +320,12 @@ func (c *ClusterResourceSet) addResourcesForControlPlane(subnetDetails *SubnetDe
 }
 
 func makeCFNTags(clusterConfig *api.ClusterConfig) []gfncfn.Tag {
-	var tags []gfncfn.Tag
+	tags := []gfncfn.Tag{
```

Comment:
Should this tag be only created when it's enabled?

Reply:
That's definitely an option here since we have to work with it not being there on stacks that would've been created before this code is in place, are you leaning more towards only tagging stacks where the prefix is disabled?

Reply:
Yeah I think we should go with that route. I think it makes it easy then to parse the value because it's enough to check if it exists or not, right?